addons/mail/static/src/models/messaging_notification_handler.js

```diff
@@ -24,6 +24,20 @@ registerModel({
          * the current users. This includes pinned channels for instance.
          */
         start() {
+            this.mailboxInfos = [
```

Comment:
Do not create non-field for storing information.

But I believe the information you are trying to store already exists, it's just the list of `threads` of the message(s), for which the thread is a mailbox.

Reply:
In the new implementation, as you have proposed, we don’t need those anymore so I have removed them.